GitDiffMargin/Git/HunkRange.cs

```diff
@@ -2,20 +2,19 @@ namespace GitDiffMargin.Git
 {
     public class HunkRange
     {
-        public HunkRange(string hunkRange, int contextLines)

```

Comment:
It was always 0, while learning the code it made sense, do you fell like could it be useful in the future?